src/EnergyPlus/WindowManager.cc

```diff
@@ -6097,7 +6097,7 @@ namespace WindowManager {
 			//  ( SurfaceWindow(SurfNum)%FrameEmis*(sigma*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin)**4 - rmir) + &
 			//    hcin*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin - tin) )
 
-			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * ( 1.0 + SurfaceWindow( SurfNum ).ProjCorrFrIn ) * ( hcin * ( SurfaceWindow( SurfNum ).FrameTempSurfIn + TKelvin - tin ) );
+			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * FrameCon * ( SurfaceWindow( SurfNum ).FrameTempSurfOut - SurfaceWindow( SurfNum ).FrameTempSurfIn );
```

Comment:
@mjwitte as far as I can tell `FrameHeatGain` is used solely for reporting purposes. The [documentation](http://bigladdersoftware.com/epx/docs/8-3/input-output-reference/group-thermal-zone-description-geometry.html#surface-window-heat-gain-energy-j) is pretty explicit that the output variables represent the total heat transfer (= conduction) through the frame, and not just the convection to the zone.

I came across this though my ASHRAE 1588RP work when I was trying to use the `WindowHeatGain` (which includes both glazing and frame) to back out a U-factor under NFRC testing conditions. The sensitivity of U-factor to frame properties was counter-intuitive and contrary to published values in ASHRAE Fundamentals Chapter 15.